public/app/features/datasources/pages/DataSourcesListPage.test.tsx

```diff
@@ -2,28 +2,28 @@ import { render, screen } from '@testing-library/react';
 import React from 'react';
 import { Provider } from 'react-redux';
 
-import { DataSourceSettings, LayoutModes } from '@grafana/data';
+import { LayoutModes } from '@grafana/data';
 import { configureStore } from 'app/store/configureStore';
-import { DataSourcesState } from 'app/types';
 
 import { navIndex, getMockDataSources } from '../__mocks__';
+import { getDataSources } from '../api';
 import { initialState } from '../state';
 
 import { DataSourcesListPage } from './DataSourcesListPage';
 
-jest.mock('app/core/services/backend_srv', () => ({
-  ...jest.requireActual('app/core/services/backend_srv'),
-  getBackendSrv: () => ({ get: jest.fn().mockResolvedValue([]) }),
+jest.mock('../api', () => ({
+  ...jest.requireActual('../api'),
+  getDataSources: jest.fn().mockResolvedValue([]),
 }));
 
-const setup = (stateOverride?: Partial<DataSourcesState>) => {
+const getDataSourcesMock = getDataSources as jest.Mock;
+
+const setup = (isSortAscending = true) => {
```

Comment:
**Nit:** what do you think about turning that parameter into an object of options? What I usually experience when these are just positional arguments is that while reading test cases I constantly need to scroll up to the function definition to understand what `setup(true)` or `setup(false)` is doing.
```suggestion
const setup = (options: { isSortAscending: boolean }) => {
```

Reply:
+1000

Reply:
Done